src/test/java/org/sagebionetworks/bridge/reporter/worker/BridgeReporterProcessorTest.java

```diff
@@ -1,5 +1,7 @@
 package org.sagebionetworks.bridge.reporter.worker;
 
+import static org.testng.AssertJUnit.assertTrue;
+import static org.testng.AssertJUnit.assertEquals;
```

Comment:
Minor: Since we're using TestNG and not JUnit, we should probably be using TestNG's native assert libraries and not the JUnit proxy. (Here and in several other places in this PR.)